src/js/helpers/helpHelper.js

```diff
@@ -89,6 +89,25 @@ const loadHistory = () => {
 	return deferred.promise;
 }
 
+const loadTechnicalHistory = () => {
+	const deferred = Q.defer();
+
+	Request.get('/help/history.md')
```

Comment:
if we're loading technical history, shouldn't it be `/help/technicalHistory.md`?